src/core/Toast/Toast.tsx

```diff
@@ -5,16 +5,17 @@
 import React from 'react';
 import { Transition } from 'react-transition-group';
 import SvgCloseSmall from '@itwin/itwinui-icons-react/cjs/icons/CloseSmall';
-import SvgStatusErrorHollow from '@itwin/itwinui-icons-react/cjs/icons/StatusErrorHollow';
-import SvgInfoCircularHollow from '@itwin/itwinui-icons-react/cjs/icons/InfoCircularHollow';
-import SvgStatusSuccessHollow from '@itwin/itwinui-icons-react/cjs/icons/StatusSuccessHollow';
 import cx from 'classnames';
 import { useTheme } from '../utils/hooks/useTheme';
 import '@itwin/itwinui-css/css/toast-notification.css';
 import { IconButton } from '../Buttons';
-import { getWindow } from '../utils/common';
+import { getWindow, StatusIconMap } from '../utils/common';
 
-export type ToastCategory = 'informational' | 'negative' | 'positive';
+export type ToastCategory =
```

Comment:
And now it comes the question about naming of these :D should be it same everywhere (status) or different as now (and exported...)